docs/learn/learn-account-advanced.md

```diff
@@ -9,6 +9,44 @@ slug: ../learn-account-advanced
 
 import RPC from "./../../components/RPC-Connection";
 
+## System Accounts
+
+:::info
+
+You can view system accounts on [Subscan](https://polkadot.subscan.io/account_list?role=module).
+
+:::
+
+As the word suggests, system accounts are used by the system. They are used, for example, for the
+treasury, as storage of crowdloan funds, and nomination pools. Those accounts they will never issue
+an [extrinsic](./learn-extrinsics.md) since they are not accounts held by users and thus do not have
```

Comment:
```suggestion
treasury, as storage of crowdloan funds, and nomination pools. These accounts will never issue
an [extrinsic](./learn-extrinsics.md) since they are not accounts held by users and thus do not have
```